.gitignore

```diff
@@ -2,3 +2,4 @@ public/
 .DS_Store
 .bundle/
 vendor/bundle/
+resources/_gen
```

Comment:
we currently check in these due to issue with Hugo on Netlify

Reply:
Got it. PR updated!